packages/manager/apps/zimbra/src/pages/dashboard/Domains/Domains.tsx

```diff
@@ -102,46 +103,57 @@ export default function Domains() {
       ),
     })) ?? [];
 
+  if (!platformUrn) {
+    return (
+      <div className="py-6 mt-8">
+        <Notifications />
+        <Outlet />
+      </div>
```

Comment:
**nitpick:** This seems to be repeated. Instead, `platformUrn` condition can be added on Line 119.